src/app/components/MediaPlayer/AudioLoader/index.jsx

```diff
@@ -32,7 +32,12 @@ const spin = keyframes`
 const Spinner = styled.svg`
   width: 64px;
   height: 64px;
+
   stroke: currentColor;
+  @media screen and (-ms-high-contrast: active) {
+    stroke: windowText;
```

Comment:
I can't actually find any clear docs for `windowText` what color is this, does it come from the user's preferences?

Reply:
I think it's just a MS specific one that gets injected - duplicated this from the BBC logo CSS, which seems to work like how we want